### ISSUE TITLE
v1.12.2 crashes when starting download of "An Audience with..." on ITV

### ISSUE THREAD

**Issue by jchhenderson** [Ascoware/get-iplayer-automator#150]
Hi all,

***What steps will reproduce the problem? Be specific, and provide as much detail as possible.***
1] Add episodes of "An Audience with..." to queue - does not matter if it is via PVR or via <cmd><g>. Note that other ITV shows download fine (at least the ones I wanted).
2] Start downloads
3] App immediately crashes with the following messages:
```
Crashed Thread:        4  Dispatch queue: NSOperationQueue 0x600000228860 (QOS: UNSPECIFIED)

Exception Type:        EXC_BAD_INSTRUCTION (SIGILL)
Exception Codes:       0x0000000000000001, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Illegal instruction: 4
Termination Reason:    Namespace SIGNAL, Code 0x4
Terminating Process:   exc handler [0]
*What is the expected output? What do you see instead?*
```

***What version of GiA are you using? On what version of OS X?***
GiA v1.12.2
MacOS Sierra v12.13.4

***Are you using a web proxy? DNS proxy? VPN? VPS?***
No, I live in the UK

***What program are you attempting to download (full title, series and episode)? **Provide only ONE program as an example.** ***
ITV PID:   L0055a0019
ITV name: "An Audience with... - Sun Mar 18"
TVDB name: "An Audience With - s01e15 - An Audience With Victoria Wood"

***Please provide any additional information below.***
Show downloads fine with Downie (if that helps)

---

[Edit: Added Downie comment]

**Comment by skovatch** [comment]
Got it. ITV threw us a curveball, with a program ID only having two parts instead of the expected 3.

**Comment by jchhenderson** [comment]
cheers - thanks for all the work you are putting into this :-)

**Comment by skovatch** [comment]
Fixed in v1.13
